tests/test_filters.py

```diff
@@ -830,15 +831,26 @@ def test_filters_sticker(self, update):
         update.message.sticker = Sticker("1", "uniq", 1, 2, False, False)
         assert filters.Sticker.ALL.check_update(update)
         assert filters.Sticker.STATIC.check_update(update)
+        assert not filters.Sticker.VIDEO.check_update(update)
+        assert not filters.Sticker.PREMIUM.check_update(update)
         update.message.sticker.is_animated = True
         assert filters.Sticker.ANIMATED.check_update(update)
         assert not filters.Sticker.VIDEO.check_update(update)
         assert not filters.Sticker.STATIC.check_update(update)
+        assert not filters.Sticker.PREMIUM.check_update(update)
         update.message.sticker.is_animated = False
         update.message.sticker.is_video = True
         assert not filters.Sticker.ANIMATED.check_update(update)
         assert not filters.Sticker.STATIC.check_update(update)
         assert filters.Sticker.VIDEO.check_update(update)
+        assert not filters.Sticker.PREMIUM.check_update(update)
+        update.message.sticker.premium_animation = File("string", "uniqueString")
+        assert not filters.Sticker.ANIMATED.check_update(update)
+        # premium stickers can probably be both, I have only see them as video thus far,
```

Comment:
they can also be animated (is_animated = True)

Reply:
I think they could also be static, as I said, doesn't really make a difference for the premium filter... Do you suggest more tests or "just" improving the comment?

Reply:
just improving the comment